src/nodejs/monitor/cli/new_transaction_alert_loop.js

```diff
@@ -125,34 +120,39 @@ const intervalObj = setInterval(() => {
 function get_last_transaction(runtimeArgs) {
   print('Searching news for ' + address)
 
-  get_last_transactions_by_address.run(runtimeArgs).then((r) => {
+  neoscan.get_address_abstracts(address, 0).then((r) => {
```

Comment:
stay consistent with camelCasing :)

Reply:
I'd prefer to leave this as is since "get_address_abstracts" matches the neoscan API convention.